tests/end-to-end/pkg/idtokenprovider/idtokenprovider.go

```diff
@@ -26,6 +25,20 @@ type dexIdTokenProvider struct {
 	config     Config
 }
 
+func GetDexTokenFromEnv() (string, error) {
```

Comment:
There is a very similar function in `tests/end-to-end/pkg/idtokenprovider/auth.go`, please adjust that aswell.